library/src/main/java/io/constructor/data/local/PreferencesHelper.kt

```diff
@@ -28,6 +28,10 @@ constructor(@ApplicationContext context: Context, prefFileName: String = PREF_FI
         get() = preferences.getString(PREF_DEFAULT_ITEM_SECTION, "")
         set(value) = preferences.edit().putString(PREF_DEFAULT_ITEM_SECTION, value).apply()
 
+    var categoriesPerTerm: Int
```

Comment:
Can we change terminology to something like `groupsShownForFirstTerm`?  That will make the code more consistent because we do not use `category` as a term in the codebase elsewhere.

Reply:
sure, will refactor